docs/src/pages/addons/writing-addons/index.md

```diff
@@ -92,6 +92,17 @@ addons.register(ADDON_ID, api => {
   });
 });
 ```
+#### Note:
```

Comment:
@Aaron-Pool is this required, or is this an issue of mis-configuration in @avihaym's project? I'm a little suspicious.

Reply:
Depends on the JSX preset/settings being used by whatever is compiling this. Theoretically `h` should be auto-injected.

Reply:
@Aaron-Pool this is an addon, so these files should be treated as react, not Vue AFAIK

Possibly related? https://github.com/storybookjs/storybook/pull/9648
cc @ndelangen

Reply:
Without this i`m getting "Uncaught ReferenceError: h is not defined"

Reply:
@avihaym I understand that this fixes the warning for you. That doesn't necessarily make it the correct fix. Adding it to the documentation makes it sound like it's the recommended workaround, but I suspect it's more likely that your project is misconfigured.

Reply:
Ok i understand... look at this issue https://github.com/storybookjs/storybook/issues/6722, He also need it.
I just want to save people time because i spend on this error a lot of time.
Thank.

Reply:
Absolutely linked to https://github.com/storybookjs/storybook/pull/9648 indeed!

As @shilman mentions, addons are rendered in react.

I might even have to remove https://github.com/storybookjs/storybook/pull/9648/files#diff-d648193349236cc3d1f8f3fb5858eaf3R10-R11 to make this fully work as expected.

Reply:
If we can get the above PR merged and released as alpha, @aviheym can test it on his project I guess, and know if further modification is required.

I'm scared of removing all user's babel config, since it's possible they are using something custom.

Perhaps I need to filter the plugin & presets to make sure there are no other JSX plugins there?

Reply:
So after playing with this hack its not really working... 
I mange to add myAddon tab to panel but the div inside <AddonPanel> not showing.
I create a new react app and its working with the same code in register.js.

So maybe just add to the docs that is not supprt Vue yet...
Thanks for your patience.

Reply:
Let's close this PR.

I hope you can assist us with testing https://github.com/storybookjs/storybook/pull/9648 @avihaym

Reply:
Let me know when you finish... i will try to help you :)

Reply:
Awesome guys 🚀

Reply:
We'll ping you when the PR is in an alpha release.

Might take a week, since @shilman is currently performing very important research in the field. 🏖